mlflow/pyfunc/model.py

```diff
@@ -461,7 +832,28 @@ def predict_stream(
         """
         Evaluates a ChatAgent input and produces a ChatAgent output. Wrapped to allow you to pass
         in a single dict with a ChatAgentRequest schema.
-        Override this function to implement a real stream prediction.
+
+        To support streaming the output of your agent, override this method in your subclass of
+        ``ChatAgent``. When implementing ``predict_stream``, keep in mind the following
+        requirements:
+
+        - Ensure your implementation adheres to the ``predict_stream`` type signature. For example,
+          streamed messages must be of the type
+          :py:class:`ChatAgentChunk <mlflow.types.agent.ChatAgentChunk>`, where each chunk contains
+          partial output from a single response message.
+        - At most one chunk in a particular response can contain the ``custom_outputs`` key. MLflow
+          utilities like ``LangGraphChatAgent`` assume this behavior, and do not have well-defined
```

Comment:
Thanks for sharing the docs preview! It makes this much easier to review - out of context, the LangGraph mention here is a bit odd, IMO it's best to remove it